makefile

```diff
@@ -30,7 +30,7 @@ deploy:
 
 # prod #########################################
 
-build: clear clean-dist install
```

Comment:
why remove `clear`? this would cause us to bloat the `/dist` folder

Reply:
@liywjl it relies on the commands being run with a $TERM set, which is not the case for Github actions. Another option is set up a custom 'lighthouse build' command